_docs-sources/guides/production-framework/ingredients/ci-cd-pipeline/ci-cd-only-path-to-prod.md

```diff
@@ -0,0 +1,5 @@
+# The CI / CD Pipeline is the only path to prod
+
+1. **The only path to prod**. The CI / CD pipeline should be the *only* way to deploy something to production. There's no other way to make changes in prod—no individual user has write access. The only exception is a small handful of admins who set up the CI / CD pipeline in the first place (bootstrapping) and then have access for in-case-of-emergency-break-glass situations. But on a daily basis, for all employees, the CI / CD pipeline is the *only* way to get something into prod. In fact, not just prod, but most shared environments too, such as stage. The only environments devs should have write access to are dev or sandbox environments designed specifically for learning about, experimenting with, and testing infrastructure in the cloud.
+2. **Locked down to your Service Catalog**. As an added measure, the CI / CD pipeline should be locked down so that it only deploys code from your company's Service Catalog. If you put these two ingredients together, your Service Catalog (which has all your vetted, tested infrastructure code) and CI / CD pipeline become the mechanisms by which you enforce all of your company's requirements (from the first section) around security, compliance, scalability, governance, etc!
```

Comment:
Do we want a sentence here saying that we can help you bootstrap your own service catalog with the stuff from _our_ service catalog (with links..). FWIW - we don't have a good introduction section for the service catalog _just yet_. But I'm working on that _this sprint_ and will likely be sourcing the ["core concepts"](https://github.com/gruntwork-io/terraform-aws-service-catalog/blob/master/core-concepts.md) section

Reply:
How Gruntwork can help is at the end.